software/glasgow/gateware/iostream.py

```diff
@@ -26,6 +26,45 @@ def _map_ioshape(direction, ioshape, fn): # actually filter+map
         name: fn(item[1]) for name, item in ioshape.items() if _filter_ioshape(direction, item)
     })
 
+class DDRBufferCanBeSimulated(io.DDRBuffer):
```

Comment:
```suggestion
class SimulatableDDRBuffer(io.DDRBuffer):
```

or just

```suggestion
class DDRBuffer(io.DDRBuffer):
```

(at your discretion).

Also, please add 2 lines between each top-level entity.

Reply:
DONE